scripts/h5py/install-h5py.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+apt-get update
+apt-get install -y libhdf5-dev
+HDF5_DIR=HDF5_DIR=/usr/lib/x86_64-linux-gnu/hdf5/serial/ python -m pip install h5py
```

Comment:
```suggestion
HDF5_DIR=/usr/lib/x86_64-linux-gnu/hdf5/serial/ python -m pip install h5py
```

Reply:
I'm hoping this is why it's not quite working at the moment 🤞

Currently rebuilding...